lib/index.js

```diff
@@ -13,6 +13,10 @@ const emojic = module.exports = {};
 
 // Append the chars to it
 iterateObject(emojis, function (value, name) {
+    
+    if (!process.stdout.isTTY || !process.platform === 'darwin')
```

Comment:
Let's support Linux too. They are not colorful, but they are still cute enough. :grin: 

Also, do we really need `process.stdout.isTTY` here? I don't see any problem piping emojis into files.